controllers/machinedeletionremediation_controller_test.go

```diff
@@ -162,12 +162,12 @@ var _ = Describe("Machine Deletion Remediation CR", func() {
 					verifyMachineNotDeleted(workerNodeMachineName)
 					verifyMachineNotDeleted(masterNodeMachineName)
 					verifyConditionsMatch([]expectedCondition{
-						{commonconditions.ProcessingType, metav1.ConditionFalse, remediationSkippedNoControllerOwner},
-						{commonconditions.SucceededType, metav1.ConditionFalse, remediationSkippedNoControllerOwner},
+						{commonconditions.ProcessingType, metav1.ConditionFalse, remediationCannotStartNoControllerOwner},
+						{commonconditions.SucceededType, metav1.ConditionFalse, remediationCannotStartNoControllerOwner},
 						// Cluster provider is not set in this test
 						{commonconditions.PermanentNodeDeletionExpectedType, metav1.ConditionUnknown, v1alpha1.MachineDeletionOnUndefinedProviderReason}})
 					verifyEvents([]expectedEvent{
-						{v1.EventTypeWarning, "RemediationSkippedNoControllerOwner", noControllerOwnerErrorMsg, true},
+						{v1.EventTypeWarning, "RemediationCannotStartNoControllerOwner", noControllerOwnerErrorMsg, true},
```

Comment:
Why don't you use the variable (remediationCannotStartNoControllerOwner)?
```suggestion
						{v1.EventTypeWarning, remediationCannotStartNoControllerOwner", noControllerOwnerErrorMsg, true},
```

Reply:
Yes, good idea, I just need to make it a string though :+1:

Reply:
I meant to just use the variable. Isn't it enough? 🤔 
FYI there are more occurrences of `"RemediationCannotStartNoControllerOwner"` that could be modified

Reply:
> I meant to just use the variable. Isn't it enough? 🤔

the variable is not a string, or did I misunderstand the question?

> FYI there are more occurrences of "RemediationCannotStartNoControllerOwner" that could be modified**

Thanks, I'll update the other after the other thread then

Reply:
You do need casting for string. [`remediationCannotStartNoControllerOwner`](https://github.com/medik8s/machine-deletion-remediation/pull/124/files#diff-e43973c0bb7cadba06a4ea1f15420fa0e4f203c1c9bea8405b0d8d8dacb4f3f3R79) is of type conditionChangeReason and I though it could be used as string for this matter, but this is wrong.